pkgs/by-name/el/elfdeps/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "elfdeps";
+  version = "0.2.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "python-wheel-build";
+    repo = "elfdeps";
+    tag = "v${version}";
+    hash = "sha256-5CrxVmtZcBYBMXw7o58CpFopYFgXD4W/S42aow1z1Xw=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+    setuptools-scm
+  ];
+
+  dependencies = [ python3Packages.pyelftools ];
+
+  nativeCheckInputs = [ python3Packages.pytestCheckHook ];
+
+  pythonImportsCheck = [
+    "elfdeps"
+  ];
+
+  preCheck = ''
+    export PATH=$PATH:$out/bin
+  '';
+
+  # tests assume that sys.executable is an ELF object
+  doCheck = !stdenv.hostPlatform.isDarwin;
```

From reading the comment, it probly makes more sense to use `isElf`
```suggestion
  doCheck = stdenv.hostPlatform.isElf;
```